master/buildbot/config.py

```diff
@@ -449,7 +451,7 @@ def load_caches(self, filename, config_dict):
             if not isinstance(caches, dict):
                 error("c['caches'] must be a dictionary")
             else:
-                valPairs = caches.items()
+                valPairs = list(iteritems(caches))
```

Comment:
iteritems(dict) return a "dict_items" type. The list is there to ensure that the result is turned into an ordinary list (which is what py2 does). 

The alternative is to use the ´´´listitems()´´´ function, which will return a list